src/core/api/checkout/checkout_ecommerce/v1/_openapi.json.tpl

```diff
@@ -994,16 +994,99 @@
               "SL"
             ],
             "description": "Requested language"
+          },
+          "details": {
+            "description": "Additional payment authorization details. Must match the correct format for the chosen payment method.",
+            "oneOf": [
+              {
+                "$ref": "#/components/schemas/PostePayAuthRequestDetails"
+              },
+              {
+                "$ref": "#/components/schemas/CardAuthRequestDetails"
```

Comment:
add a `Wallet` type with only one id field? just as a reminder that we will be managing wallets in the future 🚀

Reply:
Even though we don't have the discriminant in the underlying service spec?